projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1556,10 +1556,20 @@ public function process_form_submission() {
 				return $form->errors;
 			}
 
-			if ( ! empty( $form->attributes['salesforceData'] ) || ! empty( $form->attributes['postToUrl'] ) ) {
+			if ( ! empty( $form->attributes['salesforceData'] ) ) {
 				Post_To_Url::init();
 			}
 
+			// Deprecate postToUrl, migrate to webhooks in case someone put it to work.
```

Comment:
this deprecates the use of `postToUrl` and mirgates it into webhooks while leaving salesforce integration on its own.